Project4_election_scraper/Project4_election_scraper.py

```diff
@@ -0,0 +1,101 @@
+import requests
+from bs4 import BeautifulSoup as BS
+import csv
+
+
+def load_soup(url):
+    response = requests.get(url)
+    response.raise_for_status()
+    return BS(response.text, 'html.parser')
+
+domain = 'https://volby.cz/pls/ps2017nss/'
+district_links = []
```

Comment:
Why is `district_links` global? You are using it only in `municipality_links` so define it just there.